demos-europe/demosplan-core
# chore(refs dplan-12315): Replace draggable with sortable for places

### Ticket
DPLAN-12315

uses the new way of the dp-draggable, where the return-Value changed.

### How to test:

goto places ( `/verfahren/<procedure-id>/schritte` )
drag-drop and reload page. It should work like before


Builds on top of
https://github.com/demos-europe/demosplan-core/pull/3697
